[2024-06-18T11:28:15.851+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:28:15.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:21:00+00:00 [queued]>
[2024-06-18T11:28:15.920+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:21:00+00:00 [queued]>
[2024-06-18T11:28:15.920+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-06-18T11:28:15.945+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): skip_run> on 2024-06-18 11:21:00+00:00
[2024-06-18T11:28:15.955+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4830) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:28:15.962+0000] {standard_task_runner.py:63} INFO - Started process 4834 to run task
[2024-06-18T11:28:15.963+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'alerting_skipped', 'skip_run', 'scheduled__2024-06-18T11:21:00+00:00', '--job-id', '701', '--raw', '--subdir', 'DAGS_FOLDER/alerting_skipped.py', '--cfg-path', '/tmp/tmpjb8jbkgk']
[2024-06-18T11:28:15.965+0000] {standard_task_runner.py:91} INFO - Job 701: Subtask skip_run
[2024-06-18T11:28:16.053+0000] {task_command.py:426} INFO - Running <TaskInstance: alerting_skipped.skip_run scheduled__2024-06-18T11:21:00+00:00 [running]> on host fa5c5d972e05
[2024-06-18T11:28:16.223+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='alerting_skipped' AIRFLOW_CTX_TASK_ID='skip_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:21:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T11:21:00+00:00'
[2024-06-18T11:28:16.224+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:28:16.246+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:28:16.247+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/helper_functions.py", line 47, in alert_notif_skipped
    session = Session()
              ^^^^^^^
NameError: name 'Session' is not defined
[2024-06-18T11:28:16.260+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=alerting_skipped, task_id=skip_run, run_id=scheduled__2024-06-18T11:21:00+00:00, execution_date=20240618T112100, start_date=20240618T112815, end_date=20240618T112816
[2024-06-18T11:28:16.275+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 701 for task skip_run (name 'Session' is not defined; 4834)
[2024-06-18T11:28:16.314+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-18T11:28:16.339+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T11:28:16.341+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
